test/setup/index.ts

```diff
@@ -90,17 +87,12 @@ export async function deployMonitor(): Promise<string> {
       'snyk/kubernetes-monitor:local',
     );
 
-    console.log(`platform chosen is ${testPlatform}, createCluster===${createCluster}`);
+    console.log(`platform chosen is ${testPlatform}`);
 
     await kubectl.downloadKubectl();
     await platforms[testPlatform].setupTester();
-    if (createCluster) {
-      await platforms[testPlatform].create();
-      await platforms[testPlatform].config();
-    } else {
-      await platforms[testPlatform].config();
-      await platforms[testPlatform].clean();
-    }
+    await platforms[testPlatform].create();
+    await platforms[testPlatform].config();
```

Comment:
also here, just to make sure I understand - why did you remove the `clean` call here?

Reply:
Same as the previous comment, clean is now part of https://github.com/snyk/kubernetes-monitor/pull/395/files#diff-e5fa2adcbad65189e75788ac9a1fc12bR20-R28